src/pages/ShoppingCart/utils/calculateTotalPrice.ts

```diff
@@ -0,0 +1,25 @@
+import { Notify } from 'quasar';
+import { ShoppingCartProduct } from './concatProductsAndQuantity';
+
+export const calculateTotalPrice = (
+  cartProducts?: ShoppingCartProduct[]
+): string => {
+  const prices = cartProducts?.map((cartProduct) => {
+    const basePrice = Number(cartProduct.product.base_price_sol);
+    const quantity = Number(cartProduct.quantity);
+
+    if (isNaN(basePrice) || isNaN(quantity)) {
+      Notify.create({
+        message: `Ocurrio un error calculando el precio de ${cartProduct.product.name}, por lo que no se ha incluido en la cuenta, por favor intenta removerlo de tu carrito e incluyendolo nuevamente.`,
```

Comment:
Since I haven't the Spanish language set on the PC, I have missed adding this thing. I think maybe we can add some Spell validation to the project would be good in this case